packages/sucrase/package.json

```diff
@@ -44,14 +44,14 @@
     "jsx"
   ],
   "peerDependencies": {
-    "rollup": "^1.20.0 || ^2.0.0"
+    "rollup": "^2.53.1"
```

Comment:
We might want to keep the `||` to support older Rollup? Happy to change this line back to what it was if that would be preferable.

Reply:
I'm good with this change. We'll do a new major version.